package.json

```diff
@@ -7,7 +7,10 @@
   "browser": "./dist/contentful.browser.min.js",
   "types": "./dist/types/index.d.ts",
   "browserslist": [
-    "extends @contentful/browserslist-config"
+    "Chrome >= 79",
```

Comment:
I think we don't want the version `79` right? 🤔

Reply:
The one they mentioned was 75 so even lower. This isn’t about recommending good practices though. Just increasing compatibility for those who need it.